commercetools-models/src/main/java/io/sphere/sdk/extensions/TriggerType.java

```diff
@@ -0,0 +1,13 @@
+package io.sphere.sdk.extensions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum TriggerType {
```

Comment:
Here we should use our standard mapping and implement the `SphereEnumeration` interface, this then allows us to remove the `JsonProperty` annotatio. But requires that implement the static method `ofSphereValue` (for an example see  `StateType`).

Reply:
@katmatt Here the problem is that sphere enum starts with an uppercase, nonetheless i changed it to sphere enum by making the necessary overrides.